test/fit_tests/tests/rackhd20/test_rackhd20_api_rack_node.py

```diff
@@ -0,0 +1,160 @@
+                        if check_relation(hostnode,slavenode)== 0:
+                            continue
+                        else:
+                            return True
+                    else:
+                        return False
+    return True
+
+
+
+
+# Select nose.plugins.attrib import attr
+from nose.plugins.attrib import attr
+@attr(all=True, regression=True, smoke=True)
+
+class rackhd20_api_rack_node(fit_common.unittest.TestCase):
+    #clear the test environment
+    def tear_down(self):
+        self.test_api_delete_relation()
+        self.test_api_delete_rack()
+
+
+    def test_api_create_and_check_racks(self):
+        for operator in range(0,257):
+            if fit_common.VERBOSITY >= 8:
+                randomcharacters = '.'.join(random.sample(string.printable,8))
+                rackname = "myRack" + "_" + " " + randomcharacters + datetime.now().__str__() + " " + randomcharacters
+            else:
+                randomcharacters = '.'.join(random.sample(string.letters,8))
+                rackname = "myRack" + "_" + " " + randomcharacters + " " + randomcharacters
+            newrack = {"name" :rackname,"type":"rack"}
+            print "create new tag: " ,newrack
+            mon_url = '/api/2.0/nodes'
+            mon_data = fit_common.rackhdapi(mon_url,action='post',payload=Newrack)
+            self.assertIn(mon_data['status'],range(200,205),"Incorrect HTTP return code: {}".format(mon_data['status']))
+            rackid = mon_data['json']['id']
+            mon_url = '/api/2.0/nodes/{}'.format(rackid)
+            mon_data = fit_common.rackhdapi(mon_url)
+            self.assertIn(mon_data['status'],range(200,205),"Incorrect HTTP return code: {}".format(mon_data['status']))
+            json_node_data = mon_data['json']
+            self.assertTrue(json_node_data['name'] == newrack['name'] and json_node_data['type']=="rack","rack node field error")
+            print "query rack: " + rackname + "successfully!"
+        print "test: rack creation and query succeed!"
+
+    def test_api_delete_rack(self):
+        rack_node_list= get_node_list_by_type("rack")
+        for rack in rack_node_list:
+            mon_url = '/api/2.0/nodes/{}'.format(rack)
+            mon_data = fit_common.rackhdapi(mon_url,action='delete')
+            self.assertIn(mon_data['status'],range(200,205),"Incorrect HTTP return code: {}".format(mon_data['status']))
+
+    def test_api_add_relation(self):
+        rack_node_list= get_node_list_by_type("rack")
+        compute_node_list= get_node_list_by_type("compute")
+        switch_node_list= get_node_list_by_type("switch")
+        pdu_node_list=  get_node_list_by_type("pdu")
+        add_relation(rack_node_list[0],compute_node_list[0])
+        self.assertTrue(check_relation(rack_node_list[0],compute_node_list[0]),"Fail to check the relation")
+        #if fit_common.VERBOSITY >= 2:
+        print "compute_node_list count:",len(compute_node_list)

```

Comment:
seems like a random debug print statement.  Add verbosity back in?  or remove it.